cmd/kubeturbo/app/kubeturbo_builder.go

```diff
@@ -1030,11 +1031,14 @@ func WatchConfigMap() {
 		}
 	}
 
+	currentMinNodes := cluster.DefaultMinNodePoolSize
+	currentMaxNodes := cluster.DefaultMaxNodePoolSize
+
 	glog.V(1).Infof("Start watching the autoreload config file %s/%s", autoReloadConfigFilePath, autoReloadConfigFileName)
-	updateConfig := func() {
+	updateConfigClosure := func() {
 		newLoggingLevel := viper.GetString("logging.level")
 		currentLoggingLevel := pflag.Lookup("v").Value.String()
-		if newLoggingLevel != currentLoggingLevel {
+		if newLoggingLevel != "" && newLoggingLevel != currentLoggingLevel {
 			if newLogVInt, err := strconv.Atoi(newLoggingLevel); err != nil || newLogVInt < 0 {
 				glog.Errorf("Invalid log verbosity %v in the autoreload config file", newLoggingLevel)
 			} else {
```

Comment:
Shall we extract the logging level change into a function, e.g., `updateLoggingLevel`?